api/src/main/java/org/openmrs/OrderGroup.java

```diff
@@ -208,4 +210,26 @@ public void setId(Integer id) {
 		setOrderGroupId(id);
 	}
 	
+	/**
+	 * Gets the ParentOrderGroup to other OrderGroups, to maintain linkages between groups and
```

Comment:
Gets the parent order group, to maintain linkages between groups and support group nesting.

Reply:
@dkayiwa i have made the changes,,,kindly review